python/desc/twinkles/sprinkler.py

```diff
@@ -175,14 +175,13 @@ def sprinkle(self):
                     row['galaxyAgn_magNorm'] = np.nan
                     row['galaxyAgn_sedFilename'] = None
                     #Now insert desired Bulge properties
-                    row['galaxyBulge_sedFilename'] = self.LRG_name
+                    row['galaxyBulge_sedFilename'] = newlens['lens_sed']
                     row['galaxyBulge_redshift'] = newlens['ZLENS']
                     row['galaxyDisk_redshift'] = newlens['ZLENS']
                     row['galaxyAgn_redshift'] = newlens['ZLENS']
                     row['galaxyBulge_magNorm'] = matchBase().calcMagNorm([newlens['APMAG_I']], self.LRG, self.bandpassDict) #Changed from i band to imsimband
-                    newlens['REFF'] = 1.0 #Hard coded for now. See issue in OM10 github.
-                    row['galaxyBulge_majorAxis'] = radiansFromArcsec(newlens['REFF'])
-                    row['galaxyBulge_minorAxis'] = radiansFromArcsec(newlens['REFF'] * (1 - newlens['ELLIP']))
+                    row['galaxyBulge_majorAxis'] = radiansFromArcsec(newlens['REFF'] / np.sqrt(1 - newlens['ELLIP']))
```

Comment:
![r_eff](https://cloud.githubusercontent.com/assets/4755090/19461298/626673ae-9498-11e6-8b43-ee1c29fcefa5.png)

Above is a plot of the REFF column in the new Twinkles lens catalog. The median is at .73 so it looks like they skew smaller than 1 arcsec.